pyqtgraph/parametertree/parameterTypes.py

```diff
@@ -612,7 +612,10 @@ def __init__(self, param, depth):
         self.layout = QtGui.QHBoxLayout()
         self.layout.setContentsMargins(0, 0, 0, 0)
         self.layoutWidget.setLayout(self.layout)
-        self.button = QtGui.QPushButton(param.name())
+        title = param.opts.get('title', None)
```

Comment:
would `title = param.opts.get('title', param.name())` not work

Reply:
I was following a convention used for setting titles in other parameters, I have not tested that but it should work.